src/main/java/com/summitsync/api/grouptemplate/GroupTemplateController.java

```diff
@@ -1,4 +1,65 @@
 package com.summitsync.api.grouptemplate;
```

Comment:
Warum ist das GroupTemplate zeug hier auch drin? Sollte doch in der anderen MR sein